xinference/api/restful_api.py

```diff
@@ -1418,6 +1423,19 @@ async def stream_results():
                 self.handle_request_limit_error(e)
                 raise HTTPException(status_code=500, detail=str(e))
 
+    async def query_engines_by_model_name(self, model_name: str) -> JSONResponse:
+        try:
+            content = await (
+                await self._get_supervisor_ref()
+            ).query_engines_by_model_name(model_name)
```

Comment:
Have you ever tried if you use a http request to request this interface. What would return? 
I'm worry about the field `llm_class` cannot be serialized.